bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/scanner/RetryingReadRowsOperation.java

```diff
@@ -247,7 +250,9 @@ protected boolean isRequestRetryable() {
   @Override
   protected boolean onOK(Metadata trailers) {
     rowMerger.onCompleted();
-    completionFuture.set("");
+    if (!completionFuture.isDone()) {
```

Comment:
This is unnecessary. A future's state can only be set once

Reply:
ack.  I removed it.